gnocchi/storage/__init__.py

```diff
@@ -443,6 +447,14 @@ def compute_and_store_timeseries(self, metric, measures):
             current_first_block_timestamp = None
         else:
             current_first_block_timestamp = ts.first_block_timestamp()
+            # log if backwindow is larger than ~4MB to anticipate potential
+            # performance issue
+            if self.enable_telemetry and ts.ts.size > 320000:
+                LOG.warning("TELEMETRY: The defined backwindow size for "
```

Comment:
keep that as a warning but don't use "TELEMETRY", that's really confusing to me.

And use a sentence like:

"The defined backwindow size for metric … may negatively affect performance as it is bigger than %d (see warn_backwindow_size option)"

or something like that so it's easy for operators to get what this warning is there

Reply:
well operators can't do anything about it. this technically means nothing to them. this is just for us developers to understand how users are using Gnocchi (like how your browser/os captures metrics in background and sends it back). i'm not even sure 4MB is that bad, i just put it because i know i don't care if it's smaller than that. :)

i'm ok with putting this as an INFO log as well.

Reply:
Operators can limit the backwindow in the archive policy to limit this effect at least.

I'm good for info too.

Reply:
more specifically, i mean, i don't want to suggest to operators to limit something that may or may not be an issue. i'll be honest, i don't know if 4MB is bad or good... it's just a value i picked based on ceph benchmark tool.